### DISCUSSION TITLE
How to append images with a gutter/pad between?

### DISCUSSION THREAD

**Discussion by nikgervae** [ImageMagick/ImageMagick#4069] [Help] [answered]
I am trying to assemble two images side by side with a bit of padding between. Up to now I have been manually creating the pad image and using +append to assemble the three. But, I shouldn't have to manually create a pad, I just want to extend the canvas of the first image by 24 pixels or so and slap the second image on to the right of that, or otherwise insert a rectangle between the two images. I can't find a straightforward one-command way to do this (-border & -frame seemingly always add to both sides), but I can see several possible approaches. Any tips?

**Comment by fmw42** [comment, marked as answer]
What version of ImageMagick? What platform/OS?

For ImageMagick 7,

`magick image1 image2 -smush +24+0 result`

**Reply by fmw42** [reply]
Correction

`magick image1 image2 +smush 24 result`

See https://imagemagick.org/script/command-line-options.php#smush

**Reply by nikgervae** [reply]
That did just what I wanted, thanks! I hadn't seen -smush, and -append didn't mention it.

**Reply by Jmuccigr** [reply]
Related question: if one wanted to do a PR for these documents to, say, add a link to "smush" from "append", which is the file to work on? Is it `Website/include/command-line-options.php`?
